lib/converter.js

```diff
@@ -2,7 +2,7 @@
 const util = require('util')
 const fs = require('fs')
 const path = require('path')
-const writeFile = util.promisify(fs.writeFile)
+const writeFileAtomic = util.promisify(require('write-file-atomic'))
```

Comment:
And maybe not so soon because the [module author is departing npm](https://github.com/npm/write-file-atomic/issues/33#issuecomment-482289458) before it stops supporting Node@4 😢

Reply:
Yeah, I heard about the NPM layoffs 😞

Reply:
`write-file-atomic@3` has been release 👍